welcome/index.adoc

```diff
@@ -57,7 +57,7 @@ Next, view the xref:../microshift_release_notes/microshift-4-13-release-notes.ad
 
 * For information about Red Hat Device Edge, read the link:https://access.redhat.com/documentation/en-us/red_hat_device_edge/4/html/overview/device-edge-overview[Red Hat Device Edge overview].
 * For information about Red Hat Enterprise Linux for Edge, read link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/composing_installing_and_managing_rhel_for_edge_images/index[RHEL for Edge documentation].
-* For documentation that is not specific to {product-title}, read link:https://docs.openshift.com/container-platform/latest/welcome/index.html[{product-title} documentation].
+* For container platform documentation that is not specific to {product-title}, read link:https://docs.openshift.com/container-platform/latest/welcome/index.html[OpenShift Container Platform documentation].
```

Comment:
@ShaunaDiaz I'm confused about why the {product-title} varible would not have worked for the link text?

Reply:
@GroceryBoyJr I couldn't see an obvious reason, so I perhaps it is a distro difference?